criticality_score/generate.py

```diff
@@ -36,48 +36,47 @@
 }
 IGNORED_KEYWORDS = ['book', 'course', 'docs', 'interview', 'tutorial']
 
-
-def get_repo_urls(languages, sample_size):
+def get_repo_urls(urls, sample_size, github_lang = ''):
```

Comment:
s/get_repo_urls/get_github_repo_urls_for_language

and github_lang=None ('' is not needed).

Reply:
This one is also updated

```
def get_github_repo_urls_for_language(urls, sample_size, github_lang=None):
```